### PR TITLE
Update oam.adoc

### PR DESCRIPTION
**Pull request by jodydadescott**

### CODE REVIEW

**Comment by jodydadescott** [comment]
Updates for oam ping

**Comment by iansk** [comment]
@rcaballeromx Can you please review these changes and let me know if they're ok to merge as-is?

File: cns/saas/admin_guide/apoctl/oam.adoc [hunk 1]
```diff
@@ -2,24 +2,25 @@
...
 The `+oam+` command holds operations, administration, and maintenance tools.
 
-Examples:
-
- apoctl oam ping 5ee5a29939483e4c01a20d7b 192.168.100.101:900
-
 === `+ping+` subcommand
 
-The `+ping+` subcommand is used to run ping and print the results.
+The `+ping+` command is used for network connectivity troubleshooting. Running the command will cause the specified processing unit to initiate a flow towards the specified destination IP and port. A successful ping will return the status code `+0+` and write a report to `+stdout+`. Otherwise, the status code will be `+1+` and errors will be written to `+stderr+`. The report will contain information about the source and destination. The operation of the command and report will differ depending on whether the destination is a processing unit or not.
```
Comment (rcaballeromx) [inline cns/saas/admin_guide/apoctl/oam.adoc]:
```suggestion
Use the `+ping+` command to troubleshoot network connectivity. The command starts a flow from the specified processing unit to the destination IP and port. A successful ping returns the status code `+0+`, and writes a report to `+stdout+`. An unsuccessful ping returns the status code `+1+`, and writes a report the errors to `+stderr+`. The reports contain information about the source and destination. How the command and reports operate depends on whether the destination is a processing unit.
```

File: cns/saas/admin_guide/apoctl/oam.adoc [hunk 2]
```diff
@@ -2,24 +2,25 @@
...
 
-Running the command will notify the processing unit to initiate
-ping and wait until the corresponding results are collected. It
-also aggregates the information and prints it in a table format.
+If the destination is not a processing unit then the destination must be listening on the specified port for a successful response. The connection will be set up and then immediately dropped. The report will include detailed information about the source but only the IP and port of the destination.
```
Comment (rcaballeromx) [inline cns/saas/admin_guide/apoctl/oam.adoc]:
```suggestion
A destination that is not a processing unit must listen to the specified port for a successful response. The connection is immediately set up and then dropped. The report includes detailed information about the source, but only the IP and port of such a destination.
```

File: cns/saas/admin_guide/apoctl/oam.adoc [hunk 3]
```diff
@@ -2,24 +2,25 @@
...
 
-A successful ping will return status code `+0+` and writes the
-output in `+stdout+`. Otherwise, the status code will be `+1+` and
-the output will be in `+stderr+`.
+If the destination is a processing unit then the destination enforcer will handle the flow. Hence it is not necessary for the destination to be actively listening to the destination port. Even if it is listening on the port the enforcer will consume the flow. The report will include detailed information about both the source and destination.
```
Comment (rcaballeromx) [inline cns/saas/admin_guide/apoctl/oam.adoc]:
```suggestion
A destination that is a processing unit has an enforcer that handles the flow, and the destination doesn't have to actively listen to the port. If it is listening on the port the enforcer still consumes the flow. The report includes detailed information about both the source and destination.
```

File: cns/saas/admin_guide/apoctl/oam.adoc [hunk 4]
```diff
@@ -2,24 +2,25 @@
...
 
 Examples:
 
+ Consider the situation where there are two processing units and the admin suspects that there may be a network device within the path that is blocking traffic on port 9000. The admin knows that SSH (port 22) is working so they run the command
```
Comment (rcaballeromx) [inline cns/saas/admin_guide/apoctl/oam.adoc]:
```suggestion
 Consider the following scenario: You have two processing units and you suspect that a network device within the path connecting them is blocking traffic on port 9000. You knows that SSH on port 22 works and run the following command:
```

File: cns/saas/admin_guide/apoctl/oam.adoc [hunk 5]
```diff
@@ -2,24 +2,25 @@
...
+ 
+  ```apoctl oam ping 5ee5a29939483e4c01a20d7b 192.168.100.101:22```
+  
+ The command is successful so they run the command
```
Comment (rcaballeromx) [inline cns/saas/admin_guide/apoctl/oam.adoc]:
```suggestion
 The command succeeds, and you run the following command:
```

File: cns/saas/admin_guide/apoctl/oam.adoc [hunk 6]
```diff
@@ -2,24 +2,25 @@
...
+  
+ ```apoctl oam ping 5ee5a29939483e4c01a20d7b 192.168.100.101:9000```
+  
+ The command fails. Now the admin knows the problem is with the network.
```
Comment (rcaballeromx) [inline cns/saas/admin_guide/apoctl/oam.adoc]:
```suggestion
 Because the command fails, you now know that the problem is with the network.
```

**Review by rcaballeromx — CHANGES_REQUESTED** [review]
Thank you for your contribution!
I've left some suggestions to make the text clearer and easier to read.  PTAL

It would be great if you could add a sentence at the beginning of the example letting readers know what is the goal of the example. Why should they try to run the commands? When would they need to consider doing it?

**Comment by Pubs-MV** [comment]
Made these changes using PR#232

**Pull request closed without merging**